sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsFilterForm.java

```diff
@@ -106,8 +109,15 @@ protected void addFields() {
 		addField(FieldConfiguration.pixelSized(EventCriteria.EVENT_STATUS, 140));
 		addField(FieldConfiguration.pixelSized(EventIndexDto.DISEASE, 140));
 		addField(FieldConfiguration.withCaptionAndPixelSized(EventCriteria.REPORTING_USER_ROLE, I18nProperties.getString(Strings.reportedBy), 140));
-		ComboBox officerField = addField(FieldConfiguration.pixelSized(EventCriteria.SURVEILLANCE_OFFICER, 140));
-		officerField.addItems(fetchSurveillanceOfficersByRegion(currentUserDto().getRegion()));
+		addField(FieldConfiguration.pixelSized(EventCriteria.RESPONSIBLE_USER, 140));
+
+		ComboBox responsibleUserRoleField = addField(
```

Comment:
What exactly is the purpose of this filter?

Reply:
Well, as its name suggest, it filters on the role of the event's designated responsibleUser.
For anything else beyond that, I'm not really the one you should ask about it I think :shrug:

Reply:
Makes sense - gonna ask Carolin :)

Reply:
I removed this filter, according to the discussion in the original issue.